test/csuite/CMakeLists.txt

```diff
@@ -217,6 +217,14 @@ define_c_test(
     DEPENDS "WT_POSIX"
 )
 
+define_c_test(
```

Comment:
The definitions seem to be ordered alphabetically, you may want to put this line 190.

Reply:
Good point, fixed 89557e5